Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2098,7 +2098,7 @@ def json_to_path_generator(self, json_input, path=None):
 
 
 def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=False, metadata=None, url_keys=None,
-                    date_fields=None, json_transform_mapping=None, is_auto_json_transform=False):
+                    date_fields=None, json_transform_mapping=None, is_auto_json_transform=False, is_sorted=True):
```

Comment:
`is_sorted` is more suitable for a function name / property.
a good boolean variable name should be a verb - e.g. `sort_headers`.